build.gradle

```diff
@@ -227,12 +239,14 @@ task checkMappings {
 }
 
 task downloadIntermediary(type: Download){
-    def url = "https://github.com/FabricMC/intermediary/raw/master/mappings/${minecraft_version}.tiny"
-    src url.replaceAll(" ", "%20")
-    dest new File(cacheFilesMinecraft, "${minecraft_version}-intermediary.tiny")
+	group = buildMappingGroup
+	def url = "https://github.com/FabricMC/intermediary/raw/master/mappings/${minecraft_version}.tiny"
+	src url.replaceAll(" ", "%20")
```

Comment:
```suggestion
	src com.google.common.net.UrlEscapers.urlFragmentEscaper().escape(url)
```
This should be replaced with [Guava's UrlEscapers API](https://google.github.io/guava/releases/23.0/api/docs/com/google/common/net/UrlEscapers.html).

Reply:
Resolved. We use imports in build gradle btw.

Reply:
I included the package so you knew what to import, and I can't suggest changes for multiple arbitrary lines with the suggestion system 😜

Reply:
Ah, github was dumb as usual :trollface:

Reply:
Good idea, this was done in a rush ;)